src/components/AgreementSections/FormInfo.js

```diff
@@ -15,25 +15,34 @@ import {
 import AgreementPeriodsFieldArray from '../AgreementPeriodsFieldArray';
 import { validators } from '../utilities';
 
-export default class FormInfo extends React.Component {
+class FormInfo extends React.Component {
```

Comment:
Keep the `export default` with the `class` declaration if there's no particular to separate it (like you would need to do if you were wrapping the class in a higher-order component). 

Whenever we see a `class` lacking the `export default` it signals that it's gonna be wrapped later, which we're not doing here.

Reply:
Ah right, I did this to resolve the lint error, but I can see that a newline would have sufficed. Done now.